samples/dotnet/EdgeDownstreamDevice/Program.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using System.Security.Cryptography.X509Certificates;
+
+using Microsoft.Azure.Devices.Client;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    class Program
+    {
+
+        // 1) Obtain the connection string for your downstream device and to it
+        //    append this string GatewayHostName=<edge device hostname>;
+        // 2) The edge device hostname is the hostname set in the config.yaml of the Edge device
+        //    to which this sample will connect to.
+        //
+        // The resulting string should look like the following
+        //  "HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>;GatewayHostName=<edge device hostname>"
+        //
+        // Either set the DEVICE_CONNECTION_STRING environment variable with this connection string
+        // or set it in the Properties/launchSettings.json.
+        private static string DeviceConnectionString = Environment.GetEnvironmentVariable("DEVICE_CONNECTION_STRING");
+        private static int MESSAGE_COUNT = 10;
+        private const int TEMPERATURE_THRESHOLD = 30;
+        private static float temperature;
+        private static float humidity;
```

Comment:
You should be able to make these two static floats (`temperature` and `humidity`) local variables to `SendEvents`

Reply:
fixed.